zajecia/src/collections/Main.java

```diff
@@ -0,0 +1,241 @@
+package collections;
+
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+        zad1();
+        zad2();
+        zad3();
+        zad4();
+        zad5();
+        zad6();
+        zad7();
+        zad8();
+        zad9();
+        zad10();
+        zad11();
+        zad12();
+        zad13();
+        zad14();
+        zad15();
+    }
+
+    public static void zad1() {
+        /*
+        Zadanie 1
+        Utwórz List<String> i dodaj do niej kilka łańcuchów znaków. Następnie wyświetl zawartość listy w pętli for-each.
+         */
+        List<String> stringList = new ArrayList<>(Arrays.asList("123", "234", "345"));
+        for (String s : stringList) {
+            System.out.println(s);
+        }
+    }
+
+    public static void zad2() {
+        /*
+        Zadanie 2
+        Utwórz List<Integer> i dodaj liczby od 1 do 5. Następnie pobierz i wyświetl element na indeksie 2.
+        Usuń element o wartości 3 i wyświetl listę ponownie, aby sprawdzić czy został usunięty.
+         */
+        List<Integer> intList = new ArrayList<>(Arrays.asList(1, 2, 3, 4, 5));
+        intList.get(2);
+        intList.remove((Integer) 3);
+        intList.forEach(System.out::println);
+
+    }
+
+    public static void zad3() {
+    /*
+    Zadanie 3
+    Utwórz Set<String> i dodaj do niego kilka powtarzających się napisów, np. "Apple", "Banana", "Apple", "Cherry".
+    Wyświetl zawartość seta i sprawdź, że duplikaty nie występują.
+     */
+        Set<String> stringSet = new HashSet<>();
+        stringSet.add("Apple");
+        stringSet.add("Banana");
+        stringSet.add("Apple");
+        stringSet.add("Cherry");
+        System.out.println(stringSet);
+    }
+
+    public static void zad4() {
+        /*
+        Zadanie 4
+        Utwórz Map<String, Integer> i dodaj do niej pary klucz-wartość odpowiadające np. produktom i ich cenom. Wyświetl wszystkie klucze,
+        wszystkie wartości oraz każdą parę klucz-wartość (użyj entrySet()).
+        */
+        Map<String, Integer> map = new HashMap<>();
+        map.put("Apple", 10);
+        map.put("Banana", 12);
+        map.put("Orange", 13);
+        System.out.println(map.keySet());
+        System.out.println(map.values());
+        System.out.println(map.entrySet());
+    }
+
+    public static void zad5() {
+/*
+Zadanie 5
+Utwórz Queue<String> (np. LinkedList jako implementację), dodaj kilka elementów i symuluj działanie kolejki:
+pobierz (poll) element z kolejki i wyświetl go, a następnie pokaż stan kolejki po usunięciu elementu.
+ */
+        Queue<String> kolejka = new LinkedList<>();
+        kolejka.add("str");
+        kolejka.add("str2");
+        kolejka.add("str3");
+        kolejka.add("str4");
```

Comment:
Możesz użyć offer zamiast add, ponieważ offer jest bezpieczniejszy w przypadku, gdy kolejka osiągnie maksymalny rozmiar (choć dla implementacji LinkedList nie ma to znaczenia), ale warto miec na uwadze ze istnieje metoda offer